bash_completion.d/clush

```diff
@@ -5,7 +5,7 @@ _clush()
 {
 	# shellcheck disable=SC2034 # set/used by _init_completion
 	local cur prev words cword split
-	local word options="" skip=argv0 groupsource="" cleangroup=""
+	local word options="" compopts="" skip=argv0 groupsource="" cleangroup=""
```

Comment:
I assume we won't squash all commits when this PR will be merged? If yes, better merge this commit in the original bash_completion commit, as this is a fixup

Reply:
I avoided that because of different authors (Dominique vs. myself), but now I realized I already pushed another fixup before, so I guess it makes sense to merge this too. Will do.